irohad/main/irohad.cpp

```diff
@@ -140,6 +145,22 @@ int main(int argc, char *argv[]) {
       return EXIT_FAILURE;
     }
 
+    // check if ledger data already existing
+    auto ledger_not_empty =
+        irohad.storage->getBlockQuery()->getTopBlockHeight() != 0;
+
+    // Check if force flag to overwrite ledger is specified
+    if (ledger_not_empty) {
+        if (not FLAGS_overwrite_ledger) {
+            log->error(
+                "Block store not empty. Use '--overwrite_ledger' to force "
+                "overwrite it. Shutting down...");
+            return EXIT_FAILURE;
+        } else {
+            log->info("Overwriting existing ledger");
```

Comment:
I think that line can be omitted, because by providing such a flag you do mean such behavior. Then the code above will look quite simplier:
```cpp
if (ledger_not_empty && not FLAGS_overwrite_ledger) {
  ...
}
```